cli.py

```diff
@@ -112,41 +114,46 @@ def load_bouquets(env: str = "demo", include_private: bool = False):
     app.session.commit()
 
     universe_name = get_config_value(env, "universe_name")
-    url = f"https://{prefix}.data.gouv.fr/api/2/topics/?tag={universe_name}"
+    url = f"{base_url}/api/2/topics/?tag={universe_name}"
     if include_private:
         url = f"{url}&include_private=yes"
 
     for bouquet in iter_rel(
         {
             "href": url,
-        }
+        },
+        headers={"X-API-KEY": api_key},
     ):
         existing = app.session.query(Bouquet).filter_by(bouquet_id=bouquet["id"]).first()
         bouquet_obj = Bouquet.from_payload(bouquet)
-        bouquet_obj = upsert(app.session, bouquet_obj, existing)
-        for dataset in iter_rel(bouquet["datasets"], quiet=True):
-            dataset_obj = app.session.query(Dataset).filter_by(dataset_id=dataset["id"]).first()
+        bouquet_obj_in_db = upsert(app.session, bouquet_obj, existing)
+        for elt_id in [elt["element"]["id"] for elt in bouquet_obj._elements if elt.get("element")]:
+            dataset_obj = app.session.query(Dataset).filter_by(dataset_id=elt_id).first()
             if dataset_obj:
-                bouquet_obj.datasets.append(dataset_obj)
+                bouquet_obj_in_db.datasets.append(dataset_obj)
         app.session.commit()
 
 
-def process_dataset(env: str, d: dict, licenses: list, skip_related: bool) -> None:
-    """Process a single dataset and its resources"""
-    prefix = get_config_value(env, "prefix")
+def process_element(env: str, element: dict, licenses: list, skip_related: bool) -> None:
+    """Process a single element (dataset) and its resources"""
+    if not element.get("element", {}).get("class") == "Dataset":
+        print(f"Skipping element {element['id']} (not a dataset).")
+        return
+    base_url = get_config_value(env, "base_url")
     try:
-        if organization_id := (d.get("organization") or {}).get("id"):
+        r = requests.get(f"{base_url}/api/2/datasets/{element['element']['id']}/")
```

Comment:
`element['id']` and `element['element']['id']` is confusing. 

I think this shows confusion on how we're calling : 
1. The "element" that is the link between a topic and a resource : "element link" in the spec, but "element" here in the PR and in the API URL.
2. The "element" that is the reference to the resource : "element" in the spec, and the "element" key in the response returned by the API.

I'm hoping it's enough to clarify the variable names (and the names we use in general to talk about each of those... elements 🫣). Because changing the 'element' key in the API response doesn't sound fun... (but if needed, now is the best time to do it).